msteams-platform/get-started/build-notification-bot.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Build notification bot
+description: Sends messages in Teams channel or group chat or personal chat.
+ms.localizationpriority: medium
+ms.topic: reference
+---
+# Build notification bot
+
+The notification bot proactively sends messages in Teams channel or group chat or personal chat. You can trigger the notification bot with an HTTP request, such as cards or texts.
+
+:::image type="content" border="false" source="../assets/images/get-started/get-started-bot.png" alt-text="Illustration showing a conceptual notification bot":::
+
+In this tutorial, get familiarized with a basic Teams tab app in one of the following ways:
+
+* **Teams Toolkit codespaces**: The codespace instance allows you to experience a Teams app almost instantaneously. It opens Visual Studio Code, where Teams Toolkit extension, the app source code, and all dependencies are pre-packaged for you.
+* **Step-by-step guide**: Allows you to create a Teams app from scratch with all development environment setup.
+
+# [Teams Toolkit codespaces](#tab/teamstoolkitcodespaces)
+
+Before you create your codespace, ensure that you have:
+
+* A GitHub account to create your codespace instance
+* A [Microsoft 365 account](https://developer.microsoft.com/microsoft-365/dev-program) with sideloading permission
+* A [Microsoft 365 tenant](../concepts/build-and-test/prepare-your-o365-tenant.md)
+
+> [!TIP]
+> [GitHub codespaces](https://github.com/features/codespaces) offers the free plan up to a fixed amount of usage every month. If you need to free up the space, go to [github.com/codespaces](https://github.com/codespaces) and delete the codespaces that you no longer need.
+
+To use codespaces for creating a Teams notification bot:
+
+<a href="https://github.com/codespaces/new?hide_repo_select=true&ref=dol%2Fcodespaces&repo=348288141&machine=basicLinux32gb&devcontainer_path=.devcontainer%2Fnotification-codespaces%2Fdevcontainer.json&location=WestUs2" target="_blank"><img src="https://github.com/codespaces/badge.svg" alt="Open hello-world tab in GitHub Codespaces"></a>
```

Comment:
Use the `v3` (which is the release branch) for all codespaces samples.
Use this creation link for notification app: https://github.com/codespaces/new?hide_repo_select=true&ref=v3&repo=348288141&machine=basicLinux32gb&location=WestUs2&devcontainer_path=.devcontainer%2Fnotification-codespaces%2Fdevcontainer.json&resume=1

Reply:
Thanks @dooriya, @girliemac request you to please confirm.

Reply:
Offline discuss with @girliemac, the error is probably caused by account issue in her environment. User can create a codespace with this link either using a Microsoft account or a personal account. If using a personal account, the codespace created by user will charged by the user account rather then OfficeDev.

The message "Codespace usage for this repository is paid for by OfficeDev" won't show if using a personal account.
![image](https://github.com/MicrosoftDocs/msteams-docs/assets/10163840/fd2706ef-f883-4749-bce8-b58190a8bf41)

Reply:
Updated the codespace links as suggested.